templates/common_header.php

```diff
@@ -92,15 +92,20 @@ function setDimension1(value) {
 
 <body <?php if (array_key_exists('body_class', $values)): ?>class="<?php echo $values['body_class']; ?>"<?php endif; ?>>
 
+<?php $DISPLAY_DONATION_BANNER = FALSE; ?>
```

Comment:
If you want this to be more constant-like, I would do: `define("DISPLAY_DONATION_BANNER", false);`
and then `if (DISPLAY_DONATION_BANNER)` on the next line.
...and that leads neatly on to suggesting, why not set that `define` line in `conf/general-example`, calling it `BANNER_MESSAGE` or similar, defaulting to an empty string `""` and then print it out if present *as* the message.